DemocracyClub/yournextrepresentative
# Generate csv with 2023 modgov urls

Ref https://trello.com/c/sS43OdBJ/3343-modgov-results

This work creates a management command to parse modegov urls and council names, then open a csv with that data (to be opened and edited with election ids in google sheets). I've also updated the google sheets link for this year's elections and added new documentation for future us. 

To test: 

1. Run the management command `python manage.py resultsbot_create_mg_urls_file`.  A csv with the modgov councils and urls should be created. This is the data that has been used to populate the google spreadsheet used in the next step. 
2. Run the management command `python manage.py resultsbot_match_elections_for_mg_url`. This should start the election ID matching process with the updated 2023 google sheets url.  


```[tasklist]
### PR Checklist



Check off items once the PR addresses them.

```
